### DISCUSSION TITLE
Pixelmon

### DISCUSSION THREAD

**Discussion by HeroLeam** [PrismarineJS/mineflayer#3351] [Q&A]
Is there any way to use a bot to run on a server with a mod, for example pixelmon, in fact you wouldn't even need it to perform functions, just keep it logged in, for the farms to stay working, unless there is another way to stop loading the chunks for farms to function, plantations to grow, etc.

**Comment by Blerru** [comment]
You can use [node-minecraft-protocol-forge](https://github.com/PrismarineJS/node-minecraft-protocol-forge) to achieve this.

```js
const autoVersionForge = require('minecraft-protocol-forge').autoVersionForge;
const mineflayer = require('mineflayer');

const bot = mineflayer.createBot({ 
      // you may need to omit the version option
      // ...
});

autoVersionForge(bot._client); // _client gets node-minecraft-protocol's client
```
